examples/cookies.js

```diff
@@ -0,0 +1,50 @@
+import { check } from 'k6';
+import { browser } from 'k6/x/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+};
+
+export default async function () {
+  const page = browser.newPage();
+  const context = page.context();
+
+  try {
+    // get cookies from the browser context
+    check(context.cookies().length, {
+        'initial number of cookies should be zero': n => n === 0,
+    });
+
+    // add some cookies to the browser context
```

Comment:
Just one small comment. Would it make sense to write a test, or complement this test with cookies that are result of a "page navigation" instead of setting them through our own API? E.g.: Navigate to https://test.k6.io/my_messages.php , login and check for `sid` cookie.

Reply:
You mean like #965?

Reply:
It's a nice idea, so I added this: https://github.com/grafana/xk6-browser/pull/1005/commits/b4eeca4b3fa46ef6361af7a4f1168230b8f4ee59. Let me know what you think. Thanks!

Reply:
It's maybe not the best that it's not under the `cookies` example, but it's true that we would have to duplicate pretty much the whole `fillform` test, so should be OK :+1: